pipeline/load/display_metrics.py

```diff
@@ -0,0 +1,20 @@
+import prefect
+from prefect import task
+
+from pipeline.transform import compute_internal_matching_metrics
+from pipeline.types import Match, MatchingMetrics, MatchingOutput, User
+
+
+@task
+def display_internal_matching_metrics(metric: MatchingMetrics):
+    logger = prefect.context.get("logger")
+    logger.info("Displaying number of proposed matches per user")
+    logger.info(
+        "Matches Proposed per User:\n"
+        + "\n".join(
+            [
+                f"{count} - User {uid}"
```

Comment:
We want to print the user name, not the uid. When I wrote `User A` in the issue, I was thinking that was their name, but I wasn't clear.

So maybe you need to add users as an input to this function, use a dictionary lookup to get the name for the uid, then print that. Give that a try.